src/pkg/k8s/sa.go

```diff
@@ -33,3 +36,27 @@ func (k *K8s) SaveServiceAccount(svcAccount *corev1.ServiceAccount) (*corev1.Ser
 	metaOptions := metav1.UpdateOptions{}
```

Comment:
I know this isn't in this PR but there is current a war between "save" and "update" and currently this is the only use of "save" in the k8s package.